slack/resource_conversation.go

```diff
@@ -246,7 +312,7 @@ func resourceSlackConversationUpdate(ctx context.Context, d *schema.ResourceData
 		} else {
 			if err := client.UnArchiveConversationContext(ctx, id); err != nil {
```

Comment:
Note as above that this call does NOT work with bot tokens, only user tokens. :-(